stix_shifter_modules/secretserver/stix_transmission/status_connector.py

```diff
@@ -0,0 +1,50 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_status_connector import BaseStatusConnector
+from stix_shifter_utils.modules.base.stix_transmission.base_status_connector import Status
+from enum import Enum
+import json
+from stix_shifter_utils.utils.error_response import ErrorResponder
+
+class SecretServerStatus(Enum):
+    # WAIT, EXECUTE, SORTING, COMPLETED, CANCELED, ERROR
+    WAIT = 'WAIT'
+    EXECUTE = 'EXECUTE'
+    SORTING = 'SORTING'
+    COMPLETED = 'COMPLETED'
+    CANCELED = 'CANCELED'
+    ERROR = 'ERROR'
+
+
+class StatusConnector(BaseStatusConnector):
+    def __init__(self, api_client):
+        self.api_client = api_client
+
+    def __getStatus(self, status):
+        switcher = {
```

Comment:
does api responds with any query status? If not what's the need of this switcher dict. as a synchronous connector only status I see could be COMPLETED.